distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1740,3 +1862,140 @@ wrap_text_argument_as_value_provider val =
+   parameter. The `datelike` parameter can be a `Date`, `Date_Time`, or
+   `Time_Of_Day`.
+make_datelike_with_column_formatter : Locale -> (Date -> Text -> Text)
+make_datelike_with_column_formatter locale =
+    empty_format_uses_to_text <|
+        datelike-> format->
+            handle_illegal_argument_exception format <|
+                datelike.format format locale
+
+## PRIVATE
+   Create a `Boolean` formatter for the given format string.
+make_boolean_formatter : Text -> (Date -> Text) ! Illegal_Argument
+make_boolean_formatter format =
+    data_formatter = Data_Formatter.Value.with_format Value_Type.Boolean format
+    data_formatter.if_not_error
+        bool->
+            handle_illegal_argument_exception format <|
+                data_formatter.format bool
+
+## PRIVATE
+   Create a `Boolean` formatter that takes the format string as the second
+   parameter.
+make_boolean_with_column_formatter : (Date -> Text -> Text)
+make_boolean_with_column_formatter =
+    empty_format_uses_to_text <|
+        bool-> format->
+            data_formatter = Data_Formatter.Value.with_format Value_Type.Boolean format
+            data_formatter.if_not_error
+                data_formatter.format bool
+
+## PRIVATE
+   Create a numeric formatter for the given format string. The `number`
+   parameter can be an `Integer` or `Float`.
+make_number_formatter : Text -> Locale -> (Date -> Text)
+make_number_formatter format locale =
+    number->
+        handle_illegal_argument_exception format <|
+            number.format format locale
+
+## PRIVATE
+   Create a numeric formatter that takes the format string as the second
+   parameter. The `number` parameter can be an `Integer` or `Float`.
+make_number_with_column_formatter : Locale -> (Date -> Text -> Text)
+make_number_with_column_formatter locale =
+    empty_format_uses_to_text <|
+        number-> format->
+            handle_illegal_argument_exception format <|
+                number.format format locale
+
+## PRIVATE
+   Wrap a text formatter so that, if passed `""` or `Nothing` for the format,
+   uses .to_text instead of the normal .format method.
+empty_format_uses_to_text : (Any -> Any -> Any) -> (Any -> Any -> Any)
+empty_format_uses_to_text function = x-> format->
+    case format of
+        _ : Nothing -> x.to_text
+        "" -> x.to_text
+        _ -> function x format
+
+## PRIVATE
+   Rethrow a Java IllegalArgumentException as an Illegal_Argument.
+handle_illegal_argument_exception : Text -> Any -> Any
+handle_illegal_argument_exception format_string ~action =
+    handler cause =
+        msg = cause.payload.getMessage + ' in \"' + format_string + '\"'
+        Error.throw (Illegal_Argument.Error msg)
+    Panic.catch IllegalArgumentException handler=handler <|
+        Panic.catch UnsupportedTemporalTypeException handler=handler action
+
+## PRIVATE
+   Iterate over a range, exiting early if the body produces an `Error`.
+   TODO: Move into Range
```

Comment:
will this TODO be addressed in this PR? If not, I'd remove it or link it to an issue, otherwise will likely get lost.

Reply:
Removed.